examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -36,7 +36,7 @@
 # working with
 #
 
-import os
+from time import time
```

Comment:
Since it's used for measuring the execution time, should use [`monotonic`](https://docs.python.org/3/library/time.html#time.monotonic).